content/patterns/disclosure/examples/disclosure-image-description.html

```diff
@@ -245,61 +244,55 @@ <h3 id="id_data_label">Data for Charles Minard's Chart of Napoleon's Invasion of
       </section>
 
       <section>
-        <h2>Accessibility Features</h2>
+        <h2>アクセシビリティ機能</h2>
         <ul>
-          <li>To help people with visual impairments identify the disclosure as interactive and make it easier to perceive that clicking either the disclosure button or its label changes the expanded state, when a pointer hovers over the button or its label, the background color changes, a border appears, and the cursor changes to a pointer.</li>
+          <li>視覚障害のある人がディスクロージャーをインタラクティブな要素として識別し、ディスクロージャーボタンまたはそのラベルをクリックすることで展開状態が変わることを認識できるように、ポインターがボタンまたはそのラベルの上にあるときに背景色が変わり、境界線が表示され、カーソルがポインターに変わります。</li>
           <li>
-            Because transparent borders are visible on some systems with operating system high contrast settings enabled, transparency cannot be used to create a visual difference between the element that is focused an other elements.
-            Instead of using transparency, the focused element has a thicker border and less padding.
-            When an element receives focus, its border changes from 0 to 2 pixels and padding is reduced by 2 pixels.
-            When an element loses focus, its border changes from 2 pixels to 0 and padding is increased by 2 pixels.
+            オペレーティングシステムのハイコントラスト設定が有効になっているシステムでは透明な境界線が表示されるため、透明を使用してフォーカスされた要素と他の要素との視覚的な違いを作り出すことはできません。透明を使用せず、フォーカスされた要素は境界線が太くなりなり、パディングが少なくなります。要素がフォーカスを受け取ると、その境界線は 0 から 2 ピクセルに変わり、パディングは 2 ピクセル減少します。要素がフォーカスを失うと、その境界線は 2 ピクセルから 0 に変わり、パディングは 2 ピクセル増加します。
```

Comment:
若干、修正してみました。いかがでしょうか？

```suggestion
            オペレーティングシステムのハイコントラスト設定が有効になっているシステムでは透明な境界線が可視になるため、透明であることを用いてフォーカスされた要素と他の要素との視覚的な違いを作り出すことはできません。透明であることを用いずに、フォーカスされた要素は境界線が太くなり、パディングが少なくなります。要素がフォーカスを受け取ると、その境界線は 0 から 2 ピクセルに変わり、パディングは 2 ピクセル減少します。要素がフォーカスを失うと、その境界線は 2 ピクセルから 0 に変わり、パディングは 2 ピクセル増加します。
```